cmd/node_problem_detector.go

```diff
@@ -69,6 +70,9 @@ func main() {
 	if ke := k8sexporter.NewExporterOrDie(npdo); ke != nil {
 		exporters = append(exporters, ke)
 	}
+	if pe := prometheusexporter.NewExporterOrDie(npdo); pe != nil {
+		exporters = append(exporters, pe)
```

Comment:
Log somewhere when an exporter is successfully started. This applies to all exporters.

Reply:
Done.